crates/wasmtime/src/config.rs

```diff
@@ -316,6 +316,88 @@ impl Config {
         self
     }
 
+    /// Enables epoch-based interruption.
+    ///
+    /// When executing code in async mode, we sometimes want to
+    /// implement a form of cooperative timeslicing: long-running Wasm
+    /// guest code should periodically yield to the executor
+    /// loop. This yielding could be implemented by using "fuel" (see
+    /// [`consume_fuel`]). However, fuel instrumentation is somewhat
+    /// expensive: it modifies the compiled form of the Wasm code so
+    /// that it maintains a precise instruction count, frequently
+    /// checking this count against the remaining fuel. It would be
+    /// better to have a more lightweight mechanism.
```

Comment:
I might rephrase this from "it would be better to have..." to something focusing on how epochs are an alternative to fuel which is more lightweight. This seems to imply that fuel isn't really that useful but it's still useful for the determinism aspect (which doesn't necessarily need to be mentioned here)